src/actions/google/drive/sheets/google_sheets.ts

```diff
@@ -40,9 +42,18 @@ export class GoogleSheetsAction extends GoogleDriveAction {
 
             let filename = request.formParams.filename || request.suggestedFilename()
             if (!filename) {
+                const error: Hub.Error = {
```

Comment:
why don't we use `errorWith` here as well?

Reply:
Updated